app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughViewModel.kt

```diff
@@ -1,11 +1,17 @@
 package org.oppia.android.app.walkthrough
 
 import androidx.databinding.ObservableField
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import org.oppia.android.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 /** [ViewModel] for [WalkthroughActivity]. */
 class WalkthroughViewModel @Inject constructor() : ObservableViewModel() {
   val currentProgress = ObservableField(0)
+  val isProgressBarVisible = MutableLiveData<Boolean>(true)
```

Comment:
Best practice when using public variable and making it mutable.
Reference from `ProfileEditViewModel`

```kotlin
private val isAllowedDownloadAccessMutableLiveData = MutableLiveData<Boolean>()
val isAllowedDownloadAccess: LiveData<Boolean> = isAllowedDownloadAccessMutableLiveData
```

Reply:
Done